### PR TITLE
Article: Python Hex Function

### PR DESCRIPTION
**Pull request by nj4710**
For issue #819

### CODE REVIEW

**Comment by atjonathan** [comment]
@alayek please take a look... :smile:

File: Python-Function-Hex.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Python hex(x)
+
+`hex(x)` is a built-in function in Python 3 to convert an integer number to a lowercase [hexadecimal](https://www.mathsisfun.com/hexadecimals.html) string prefixed with “0x”.
+
+## Argument
+
+This function takes one argument `x` that should be of integer type.
```
Comment (atjonathan) [inline Python-Function-Hex.md]: ~~argument `x` that~~ argument, `x`, that

**Comment by atjonathan** [comment]
@alayek Just need a quick `QA` from you :laughing:

File: Python-Function-Hex.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+This function takes one argument, `x`, that should be of integer type.
+
+## Return
+
+This function returns a lowercase hexadecimal string prefixed with "0x".
+
+## Example
+
+```python
+print hex(16)   # prints  0x10
```
Comment (alayek) [inline Python-Function-Hex.md]: Getting this on REPL
```python
Traceback (most recent call last):
  File "python", line 1
    print hex(16)
            ^
SyntaxError: invalid syntax
```

Comment (alayek) [inline Python-Function-Hex.md]: You are using Python 2 print syntax, use `print()`

**Comment by Rafase282** [comment]
[This](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/Python-Functions) is the main article, which refers to other articles on Python functions.

Please edit this file as well and include yours, in alphabetical order.

**Comment by nj4710** [comment]
@Rafase282  done. any more changes?

**Comment by alayek** [comment]
# LGTM :+1:  

Let's :shipit: :package:

**Comment by Rafase282** [comment]
@nj4710 you haven't add it yet.

**Comment by nj4710** [comment]
I have created a pull request

**Comment by Rafase282** [comment]
# LGTM

**Pull request merged**